src/CreatorToken.sol

```diff
@@ -88,6 +91,8 @@ contract CreatorToken is ERC721 {
     creatorTokenURI = _tokenURI;
     creator = _creator;
     CREATOR_FEE_BIPS = _creatorFee;
+    CREATOR_ROYALTY_BIPS = _creatorRoyalty;
```

Comment:
I think it would make sense to also add a check to make sure the royalty is less than the max fee too, again, just as a sanity check. Would want a test for this too.

Reply:
Sounds good, added the check and test here: [4ff6b9e](https://github.com/showtime-xyz/creator-tokens/pull/34/commits/4ff6b9ec6f6ccf63eb5827de5216bca924287d5c)